src/FixedDataTable.js

```diff
@@ -569,15 +570,7 @@ class FixedDataTable extends React.Component {
   }
 
   componentWillReceiveProps(/*object*/ nextProps) {
-
```

Comment:
The earlier way of checking of controlled scrolling through `scrollTop` and `scrollLeft` was a bit flawed.
`scrollTop` and `scrollLeft` can be different without any apparent scroll (something like scrollTop going from 100 to 200 units, when the table was just only 50units long), but will lead to `_didScrollStart` being fired because of the equality check).

Now since we use the reducers to check for these, they won't give false positives.